pandas/io/gbq.py

```diff
@@ -682,6 +700,17 @@ def read_gbq(query, project_id=None, index_col=None, col_order=None,
 
         .. versionadded:: 0.19.0
 
+    **kwargs : Arbitrary keyword arguments
+        config (dict): query config parameters for job processing.
+        For example:
+
+            config = {'query': {'useQueryCache': False}}
+
+        For more information see `BigQuery SQL Reference
+            <https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#configuration.query>`
```

Comment:
no indentation relative to "For more ...) is needed here (otherwise possibly will give errors when building the docs)